webapp/src/components/backstage/toast_banner.tsx

```diff
@@ -0,0 +1,154 @@
+import React, {useState, useEffect} from 'react';
+import styled from 'styled-components';
+import {CSSTransition, TransitionGroup} from 'react-transition-group';
+
+const Ctx = React.createContext({} as ToastFuncs);
+
+const StyledToast = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 4px 4px 4px 12px;
+    margin: 4px;
+    height: 40px;
+
+    background: var(--center-channel-color);
+    box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.12);
+    border-radius: 4px;
+
+    &.fade-enter {
+        transform: translateY(80px);
+    }
+  
+    &.fade-enter-active {
+        transform: translateY(0);
+        transition: transform 500ms linear;
+    }
+  
+    &.fade-exit {
+        transform: translateY(0);
+    }
+  
+    &.fade-exit-active {
+        transform: translateY(280px);
+        transition: transform 1s cubic-bezier(1, -0.4, 0.5, 1);
+    }
```

Comment:
I agree linear on the entrance probably isn't the best, but not a huge deal. Here's a possible alt. set of timings.

https://user-images.githubusercontent.com/11724372/152876957-9714ac9e-8e5d-4a84-bd66-3c6359e6a8b1.mp4

```suggestion
    &.fade-enter {
        transform: translateY(80px);
    }

    &.fade-enter-active {
        transform: translateY(0);
        transition: transform 0.5s cubic-bezier(.44,.13,.42,1.43)
    }

    &.fade-exit {
        transform: translateY(0);
    }

    &.fade-exit-active {
        transform: translateY(280px);
        transition: transform 0.75s cubic-bezier(.59,-0.23,.42,1.43);
    }
```